backend/server.js

```diff
@@ -49,7 +52,12 @@ app.get('/', (req, res, next) => {
 app.get('/login', async (req, res, next) => {
   try {
     if (req.session.email) {
-      res.send(req.session.email);
+      const user = await User.findOne({
+        where: {
+          email: req.session.email,
+        }
+      })
+      res.send(user);
     }
```

Comment:
add an else{} and res.send an response if session does not have an email - will break if route is called